server/modules/module.js

```diff
@@ -19,11 +19,11 @@ import * as wallGeometry from '../util/wall_geometry.js';
 import * as moduleTicker from './module_ticker.js';
 import assert from '../../lib/assert.js';
 import library from './module_library.js';
-import network from '../network/network.js';
+import * as network from '../network/network.js';
 import {StateManager} from '../state/state_manager.js';
 import {delay} from '../../lib/promise.js';
 import {getGeo} from '../util/wall_geometry.js';
-import {clients} from '../network/clients.js';
+import {clients} from '../network/network.js';
```

Comment:
It this kind of double-import an idiom now, or should it be const clients = network.clients?

Reply:
"Destructuring" import is the right way to do it.